Cron/Order.php

```diff
@@ -190,10 +190,21 @@ protected function processOrderStatusUpdate(OrderInterface $order, string $newSt
     protected function cancelOrder(OrderInterface $order) : bool
     {
         try {
+            $isUnholded = false;
+            if ($order->canUnhold()) {
```

Comment:
I assume this returns true if the order is not holded in the first place?

Reply:
@crfroehlich If this returns true then the order is holded in the first place therefore it must be unholded

Reply:
`canUnhold()` _only_ returns `true` [if the order is holded](https://github.com/magento/magento2/blob/2.3/app/code/Magento/Sales/Model/Order.php#L763).

Reply:
so don't we always want to cancel if the order can be cancelled?

Reply:
We do want to cancel it if it can be canceled thus we need to unhold it then cancel it @crfroehlich

Reply:
Nevermind. Operator error.

Reply:
Coolness, merging